### PR TITLE
registering demoparticipant

### PR DESCRIPTION
**Pull request by SUSMOY97**
Hey Maintainer !

This is really great . Kindly accept the request

### CODE REVIEW

File: content/participants/ttjc@SUSMOY97/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: "SUSMOYOP"
+type: "participant"
+name: "SUSMOY MUKHERJEE"
+portfolio: "https://tanaypratap.com"
```
Comment (gkanishk) [inline content/participants/ttjc@SUSMOY97/index.md:5]: please provide your portfolio or leave it blank

File: content/participants/ttjc@SUSMOY97/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+I can eat 3 packets of Biriyani at a time XD.
+
+### help I need?
+
+I am making various projects for this challenge on react as well as learning react everyday , so i might need help with react-router.
+
+### how to contact me?
+
+- Twitter
```
Comment (gkanishk) [inline content/participants/ttjc@SUSMOY97/index.md:32]: provide a link to your social handle or remove this. also suggestion use markdown to sinplify things

**Comment by gkanishk** [comment]
please resolve this issue else we need to close the **PR**

File: content/participants/ttjc@SUSMOY97/index.md (hunk 2 above)
Comment (SUSMOY97) [reply content/participants/ttjc@SUSMOY97/index.md]: Done

**Comment by SUSMOY97** [comment]
is it ok ?

**Pull request closed without merging**